.env.example

```
@@ -17,7 +16,8 @@ SESSION_SIGNING_KEY_CURRENT=
 SESSION_SIGNING_KEY_PREVIOUS=
 FIELD_ENCRYPTION_KEY_CURRENT=
 FIELD_ENCRYPTION_KEY_PREVIOUS=
-RECEIPT_ANALYZER_PROVIDER=workers_ai
+RECEIPT_ANALYZER_PROVIDER=taggun
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECEIPT_ANALYZER_PROVIDER key should go before the SESSION_SIGNING_KEY_CURRENT key

(UnorderedKey)